src/main/java/org/broadinstitute/consent/http/resources/ElectionReviewResource.java

```diff
@@ -66,7 +66,7 @@ public ElectionReview getElectionReviewByElectionId(@PathParam("electionId") Int
     @Path("access/{electionId}")
     @Produces("application/json")
     @RolesAllowed({ADMIN, MEMBER, CHAIRPERSON, ALUMNI})
-    public ElectionReview getAccessElectionReviewByReferenceId(@PathParam("electionId") Integer electionId, @QueryParam("isFinalAccess") Boolean isFinalAccess) {
+    public ElectionReview getAccessElectionReviewByReferenceId(@PathParam("electionId") Integer electionId) {
```

Comment:
This changes the api signature which would require a change to the api-docs as well. This method is called from the UI and provides the `isFinalAccess` as well so this PR will require a companion on the UI side.

Reply:
Here and below, to make this more backwards compatible, leave the query param in place so that the method signature doesn't change, but instead just ignore the query param. That way, UI code can change separately from this and either call this endpoint with or without the query param. In a follow-up PR, we can remove the unused query param. That maintains backward compatibility and allows for the back end and the front end to be released independently.